From 50ade7cf8fee24649bc216a22eb60d11ef1a4b31 Mon Sep 17 00:00:00 2001
From: Ambjoern Shield 🛡 <sekhmet396gripen@gmail.com>
Date: Thu, 10 Dec 2020 19:50:58 +0100
Subject: [PATCH] EmuoFkWCRO9DtqTeH gitignore icloud DB

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index a3fc553..990667b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -102,3 +102,5 @@
 
 !/Download-on-the-App-Store/*
 
+!/Download-on-the-App-Store/*
+
--
Working Copy 4.5.0

